lib/cre8or.ts

```diff
@@ -24,3 +27,21 @@ export const getLockedCount = async (address: string) => {
 
   return count
 }
+
+export const transferCre8orToSmartWallet = async (
+  contractAddress: string,
+  from: string,
+  to: string,
+  tokenId: number,
+  signer: Signer,
```

Comment:
move this `transferErc721` function into a `useErc721TransferHook` so you don't need to pass `signer` as a param 🧹

Reply:
```suggestion
```

remove as param and move to hook where it can use `useSigner` hook directly

Reply:
updated this 🔨